test/quantization/test_quantization.py

```diff
@@ -575,7 +575,6 @@ def checkQuantized(model):
         model = quantize_dynamic(NestedModel().eval(), qconfig_dict)
         checkQuantized(model)
 
-    @unittest.skip("temporarily disable the test")
     @given(qengine=st.sampled_from(("fbgemm",)))
```

Comment:
@raghuramank100 could you add `qnnpack` here as well? I believe it was skipped for the same reason.